### PR TITLE
Remove enableUiTheming FF logic

### PR DESCRIPTION
**Pull request by scavnickyj**
Theming will be enabled for all tiers by default and FF will be removed.

JIRA: TNT-692

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Review by pbenes — APPROVED** [review]

File: libs/sdk-backend-bear/src/backend/workspace/styling/styling.ts [hunk 1]
```diff
@@ -4,71 +4,33 @@ import { IColorPaletteItem, ITheme } from "@gooddata/sdk-model";
...
-                      .catch((err) => {
-                          if (isApiResponseError(err)) {
-                              return {};
-                          }
-
-                          throw err;
-                      }),
-              )
-            : {};
+        const identifier = config.selectedUiTheme as string;
```
Comment (ivan-nejezchleb) [inline libs/sdk-backend-bear/src/backend/workspace/styling/styling.ts:18]: what will happen when selectedUiTheme is undefined, or empty string?
Before it was handled by isThemable

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2609

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2370/ : SUCCESS in 1m 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2664/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1482/ : SUCCESS in 24m 31s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10110/ : SUCCESS in 13m 39s

**Pull request merged**